modules/exploits/windows/http/geutebrueck_gcore_x64_rce_bo.rb

```diff
@@ -0,0 +1,286 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'nokogiri'
+require 'open-uri'
+
+class MetasploitModule < Msf::Exploit::Remote
+    include Msf::Exploit::Remote::Tcp
+
+    Rank = NormalRanking
+
+    def initialize(info = {})
+        super(update_info(info,
+                          'Name' => 'Geutebrueck GCore - GCoreServer.exe Buffer Overflow RCE',
+                          'Description'	=> 'This module exploits a stack Buffer Overflow in the GCore server (GCoreServer.exe). The vulnerable webserver is running on Port 13003 and Port 13004, does not require authentication and affects all versions from 2003 till July 2016 (Version 1.4.YYYYY).',
+                          'License' => MSF_LICENSE,
+                          'Author' =>
+                          [
+                              'Luca Cappiello',
+                              'Maurice Popp'
+                          ],
+                          'References' =>
+                          [
+                              ['www.geutebrueck.com', '']
```

Comment:
Invalid format. Please use:

```ruby
['EBD', '41153'],
['URL', 'https://packetstormsecurity.com/files/140762/geutebrueck-overflow.rb.txt']
```

And add any other references to the advisory, PoC or exploit if available.